styles/index.css

```diff
@@ -0,0 +1,174 @@
+@import url('https://fonts.googleapis.com/css2?family=Bentham&family=Indie+Flower&family=Space+Grotesk&display=swap');
+* {
+    font-family: 'Bentham', serif;
+    font-family: 'Indie Flower', cursive;
+    font-family: 'Space Grotesk', sans-serif;
+}
+
+body {
+    background-color:rgb(9, 124, 255);
+    
+    opacity: 0.8;
+}
+#temp-count {
+    color: pink;
+    font-size: 2em;
+}
+
+#temp-count.red {color: red;}
+#temp-count.orange {color: orange;}
+#temp-count.yellow {color: gold;}
+#temp-count.green {color: green;}
+#temp-count.teal {color: teal;}
```

Comment:
Collapsing these rules into one line works, but I usually see the CSS rules expanded out (like L30-32) for readability!